lib/model/QubitDigitalObject.php

```diff
@@ -1264,13 +1264,26 @@ public function delete($connection = null)
     if (!empty($this->informationObjectId))
     {
       QubitSearch::getInstance()->update($this->getInformationObject());
+      $this->deleteFromAssociatedInformationObject();
```

Comment:
I'd do it before updating the search index, just in case it tries to get some DO data in there.